src/doc/rustdoc/src/advanced-features.md

```diff
@@ -80,7 +80,8 @@ pub struct BigX;
 Then, when looking for it through the `rustdoc` search, if you enter "x" or
 "big", search will show the `BigX` struct first.
 
-There are some limitations on the doc alias names though: you can't use `"` or whitespace.
+There are some limitations on the doc alias names though: you can't use quotes (`'`, `"`) or whitespaces;
+ASCII spaces are accepted as long as they're not at the beginning or the end of the alias.
```

Comment:
```suggestion
There are some limitations on the doc alias names though: they cannot contain quotes (`'`, `"`)
or non-ASCII whitespace. ASCII whitespace is allowed if it does not start or end the alias.
```

Maybe something like this? Because as-is, saying whitespace isn't allowed and then saying it _is_ (sometimes) allowed is a bit contradictory :)

Reply:
Isn't it plural?
> [...] or non-ASCII whitespaces. ASCII whitespaces are allowed if they don't [...]

Bikeshedding but `\t` are ASCII whitespaces, another wording may be:
> [aliases] cannot contain quotes (`'`, `"`) or non-` ` whitespaces (e.g. `\t`, NBSP). ASCII spaces are allowed if they don't start or end the alias.

But screen readers may just skip the ` ` and blatantly say “non-whitespaces.”

Yet another wording:
> [aliases] cannot contain quotes (`'`, `"`), start or end with a space, and all spaces must be ASCII (i.e. the other whitespaces aren't allowed).

Reply:
> Isn't it plural?

AIUI "whitespace" is usually uncountable, so it can be used as either singular or plural without a "s" ("whitespaces" does seem to exist, just less common). Or "whitespace characters" if you need it countable.

Everything else form your suggestions sound good to me though. Or just something like "... cannot contain quotes (`'`, `"`) or most whitespace. ASCII spaces are allowed ..."

Reply:
Looks neat; co-authored you as it's basically your sentence, not sure if it's appropriate.

Reply:
Never hurts to ask, but none needed here; I'll let you use my five words without credit :)